versionedRouter.js

```diff
@@ -114,7 +118,7 @@ if (startDestTransformer) {
 
       const transformedEvents = [];
       await Promise.all(
-        Object.entries(groupedEvents).map(async ([dest, destEvents]) => {
+        Object.entries(groupedEvents).map(async ([destEvents]) => {
```

Comment:
This needs to be fixed. The first argument will be destID and the second is list of events.

Example here: https://repl.it/repls/SmoggyNextCrash

Reply:
Done